modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts

```diff
@@ -43,6 +44,36 @@ test('Assert table column labels', async ({dataSetsPage, page}) => {
 	await dataSetsPage.deleteDataSet();
 });
 
+test('Assert table cell content', async ({dataSetsPage, page}) => {
+	const dataSetConfig = {
+		name: DEFAULT_LABEL.DATA_SET,
+		restApplication: '/data-set-manager/fields',
+		restEndpoint: '/',
+		restSchema: 'FDSField',
+	};
+
+	await dataSetsPage.goto();
+	await dataSetsPage.createDataSet(dataSetConfig);
```

Comment:
We can now use creation and deletion data [with API](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/helpers/DataSetManagerApiHelpers.ts). It's much faster, less prone to flakiness, and easier to maintain compared to relying on UI.

Reply:
I've consider the creation of the data set via API, but it also makes sense testing the UI part.
Right now there are 3 tests that create/delete the dataset. We could refactor this and create just a couple of test that test each approach:
- everything via api (columns, content, actions)
- everything via UI (columns, content, actions)